briareus-local/src/main/java/moe/dare/briareus/local/LocalJvmProcess.java

```diff
@@ -34,11 +35,27 @@ public boolean isAlive() {
 
     @Override
     public OptionalInt exitCode() {
-        return process.isAlive()? OptionalInt.empty() : OptionalInt.of(process.exitValue());
+        return process.isAlive() ? OptionalInt.empty() : OptionalInt.of(process.exitValue());
     }
 
     @Override
     public CompletionStage<RemoteJvmProcess> onExit() {
         return terminatedFuture.thenApply(any -> this);
     }
+
+    /**
+     * @return pid of process or name of this virtual machine.
+     */
+    @Override
+    public Object getExternalId() {
```

Comment:
see https://stackoverflow.com/questions/4750470/how-to-get-pid-of-process-ive-just-started-within-java-program
PID of #proccess required =(